### PR TITLE
CogniSwitch Agent Toolkit for LangChain

### PR DESCRIPTION
**Pull request by CogniJT**
**Description**: CogniSwitch focusses on making GenAI usage more reliable. It abstracts out the complexity & decision making required for tuning processing, storage & retrieval. Using simple APIs documents / URLs can be processed into a Knowledge Graph that can then be used to answer questions.

**Dependencies**: No dependencies. Just network calls & API key required
**Tag maintainer**: @hwchase17
**Twitter handle**: https://github.com/CogniSwitch
**Documentation**: Please check `docs/docs/integrations/toolkits/cogniswitch.ipynb`
**Tests**: The usual tool & toolkits tests using `test_imports.py`

PR has passed linting and testing before this submission.

### CODE REVIEW

**Comment by CogniJT** [comment]
@hwchase17 @baskaryan Please take a look at this

**Comment by saiCogniswitch** [comment]
hi @rlancemartin  Good morning
we have updated the files according to the lint issues shown, can you check and let us know if any changes are required.
Thank you,
have a good day

**Comment by saiCogniswitch** [comment]
hi @rlancemartin 
the checks have been passed, except for vercel deployment, let us know how we can proceed forward.
Thanks,
have a good day

**Comment by CogniJT** [comment]
If we could get Vercel access, we could debug the issue further. Might be a simpler change given that the team is busy.
We had actually started a PR on Sep 21, 2023. This is just a continuation of that. Request you to please prioritize this.

File: docs/docs/integrations/toolkits/cogniswitch.ipynb
Comment (baskaryan) [inline docs/docs/integrations/toolkits/cogniswitch.ipynb:1]: can we get rid of the html tags? those are causing the vercel issues

Comment (saiCogniswitch) [reply docs/docs/integrations/toolkits/cogniswitch.ipynb:1]: sure, I will update the notebook and put it back there.

**Comment by saiCogniswitch** [comment]
hi @baskaryan, We have updated the notebook and the vercel deployment is also successful, awaiting your approval.
Thanks,
have a good day

**Review by baskaryan — APPROVED** [review]

**Comment by saiCogniswitch** [comment]
Thanks @baskaryan  for the approval and merge of the PR.
have a good day.

**Pull request merged**